File: Facade Pattern C# version/FacadeConsole/Projector.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FacadeConsole
{
    public class Projector
    {
        DvdPlayer dvdPlayer;
        bool active = false;

        public void TurnOn()
        {
            active = true;
        }

        public void TurnOff()
        {
            active = false;
        }

        public void TvMode()
        {
            Console.WriteLine("You are on Tv mode.\n");
        }

        public void WideScreenMode()
        {
            Console.WriteLine("Wide screen mode has been turned on.\n");
        }
    }
}

File: Facade-Testing/src/Projector.java
public class Projector {
    DvdPlayer dvdPlayer;
    boolean on = false;

    public void on(){
        this.on = true;
    }

    public void off(){
        this.on = false;
    }

    public void tvMode(){
        System.out.println("You are on Tv mode.\n");
    }

    public void wideScreenMode(){
        System.out.println("Wide screen mode has been turned on.\n");
    }
}
